e2e-tests/insurance/cypress/e2e/journeys/your-buyer/trading-history/trading-history-page.spec.js

```diff
@@ -94,6 +93,60 @@ context('Insurance - Your Buyer - Trading history page - As an exporter, I want
       });
     });
 
+    describe(`${AMOUNT_OVERDUE} and ${TOTAL_OUTSTANDING}`, () => {
+      describe(`when not selecting a ${OUTSTANDING_PAYMENTS} radio`, () => {
+        it('should not render a heading', () => {
+          field(TOTAL_OUTSTANDING).heading().should('not.be.visible');
+        });
+
+        it(`should not render a label for ${TOTAL_OUTSTANDING}`, () => {
+          field(TOTAL_OUTSTANDING).label().should('not.be.visible');
+        });
+
+        it(`should not render an input for ${TOTAL_OUTSTANDING}`, () => {
+          field(TOTAL_OUTSTANDING).input().should('not.be.visible');
+        });
+
+        it(`should not render a label for ${AMOUNT_OVERDUE}`, () => {
+          field(AMOUNT_OVERDUE).label().should('not.be.visible');
+        });
+
+        it(`should not render an input for ${AMOUNT_OVERDUE}`, () => {
+          field(AMOUNT_OVERDUE).input().should('not.be.visible');
```

Comment:
Should all of these tests be `not.exist`? I think visible would still pass if the element exists in the DOM.

Reply:
the other way around :P `not.exist` fails as it exists in the dom but `not.be.visible` works

Reply:
Ah gotcha, sorry I was thinking of the design scenario meaning it shouldn't exist on the page but this is for conditional reveals so make sense 👍